spec/sidekiq/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -473,34 +466,6 @@ def expect_retryable_error(error_class)
           expect(Form526JobStatus.last.submission.submitted_claim_id).to eq(12_345_678)
         end
       end
-
-      context 'with toxic exposure Flipper enabled for user' do
-        let(:submission) do
-          create(:form526_submission,
-                 :with_everything,
-                 user_uuid: user.uuid,
-                 auth_headers_json: auth_headers.to_json,
-                 saved_claim_id: saved_claim.id,
-                 submit_endpoint: 'claims_api')
-        end
-
-        before do
-          Flipper.enable(:disability_526_toxic_exposure, user)
-          allow_any_instance_of(Auth::ClientCredentials::Service).to receive(:get_token).and_return('fake_access_token')
-        end
-
-        after do
-          Flipper.disable(:disability_526_toxic_exposure, user)
-        end
-
-        it 'performs a successful submission' do
```

Comment:
whoa, hold on. we still need this test.
having the `submit_endpoint: 'claims_api'` will make this pass, right? 🤔 
we just don't need the flipper enabling/disabling parts